run

```diff
@@ -1,22 +1,25 @@
 #/bin/bash
 
-set -e 
+set -x
 
 SERVICES=$*
-if [[ -z "$SERVICES" ]]; then
-  SERVICES="regalions" # developer regalions
-  echo "Not building any services"
+if [[ -n "$SERVICES" ]]; then
+  echo "building services: $SERVICES"
+  SERVICES="-b $SERVICES" # developer regalions
+  echo "building services: $SERVICES"
```

Comment:
Should this be "built"?

Reply:
Second one should not be outputted anyway... skaffold trips the docker build. Removing.